[2023-10-28 00:37:43,397] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:37:43,414] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:37:43] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:37:44,726] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:37:44,729] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:37:44,829] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:37:44] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 00:57:53,985] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 00:57:53,992] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 00:57:53,996] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 00:57:53] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:04:50,335] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 01:04:50,341] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 01:04:50,344] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:04:50] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:06:27,412] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 01:06:27,416] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 01:06:27,418] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:06:27] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:08:38,868] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 01:08:38,874] DEBUG get_fornecedores() L80   3 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 01:08:38,878] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:08:38] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:08:46,288] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'teste' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:08:46,309] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'teste' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 01:08:46,312] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:08:46] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:09:07,269] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'Deposito' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:09:07,286] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'Deposito' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 01:09:07,290] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:09:07] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:10:06,693] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 01:10:06,698] DEBUG get_fornecedores() L80   5 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 01:10:06,700] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:10:06] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:10:53,935] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'teste' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:10:53,959] WARNING add_fornecedor() L53   Erro ao adicionar fornecedor 'teste', Fornecedor de mesmo nome já salvo na base :/ - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L53  
[2023-10-28 01:10:53,961] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:10:53] "[31m[1mPOST /fornecedor HTTP/1.1[0m" 409 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:12:06,807] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'teste' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:12:06,814] WARNING add_fornecedor() L53   Erro ao adicionar fornecedor 'teste', Fornecedor de mesmo nome já salvo na base :/ - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L53  
[2023-10-28 01:12:06,821] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:12:06] "[31m[1mPOST /fornecedor HTTP/1.1[0m" 409 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:12:12,968] DEBUG get_fornecedores() L70   Coletando fornecedores  - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L70  
[2023-10-28 01:12:12,975] DEBUG get_fornecedores() L80   5 fornecedores encontrados - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L80  
[2023-10-28 01:12:12,978] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:12:12] "GET /fornecedores HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:12:30,813] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'teste1' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:12:30,839] DEBUG add_fornecedor() L47   Adicionado fornecedor de nome: 'teste1' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L47  
[2023-10-28 01:12:30,845] INFO _log() L224  127.0.0.1 - - [28/Oct/2023 01:12:30] "POST /fornecedor HTTP/1.1" 200 - - call_trace=C:\Users\lab\anaconda3\Lib\site-packages\werkzeug\_internal.py L224 
[2023-10-28 01:12:55,328] DEBUG add_fornecedor() L39   Adicionando fornecedor de nome: 'teste' - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L39  
[2023-10-28 01:12:55,334] WARNING add_fornecedor() L53   Erro ao adicionar fornecedor 'teste', Fornecedor de mesmo nome já salvo na base :/ - call_trace=C:\dev\pyprojects\devfull-mvp1\app_api\app.py L53  
